extensions/blocks/calendly/edit.js

```diff
@@ -53,17 +62,13 @@ export default function CalendlyEdit( props ) {
 		url,
 	} = validatedAttributes;
 	const [ embedCode, setEmbedCode ] = useState( '' );
-	const [ notice, setNotice ] = useState();
 
-	const setErrorNotice = () =>
-		setNotice(
-			<>
-				{ __(
-					"Your calendar couldn't be embedded. Please double check your URL or code.",
-					'jetpack'
-				) }
-			</>
+	const setErrorNotice = () => {
```

Comment:
As I noted here, I wonder if we should extract this into a shared function maybe?

https://github.com/Automattic/jetpack/pull/14895#discussion_r388372281

Reply:
I think this is a good idea, but lets do it in another PR. I have created an issue to track it here: https://github.com/Automattic/jetpack/issues/14899